webpack_config/production.js

```diff
@@ -79,6 +67,33 @@ module.exports = merge.smart(common, {
       enabled: true
     }),
 
+    new PreloadWebpackPlugin({
+      rel: 'preload',
+      as(entry) {
+        if (/\.(woff|woff2)$/.test(entry)) return 'font';
+        return 'script';
+      },
+      include: 'allAssets',
+      fileWhitelist: [/\.(woff|woff2)$/]
+    }),
+
+    new PreloadWebpackPlugin({
+      rel: 'preload',
+      include: 'allAssets',
+      fileWhitelist: [/\.worker\.js$/],
+      crossorigin() {
+        return 'anonymous';
+      }
+    }),
+
+    new CompressionPlugin({
+      filename: '[path].gz',
+      algorithm: 'gzip',
+      test: /\.js$|\.css$|\.html$/,
+      threshold: 10240,
+      minRatio: 0.8
+    }),
```

Comment:
Does it make a difference if the gzip is done by the build process instead of the server?

Reply:
Not really, I am just testing locally with this, just so Lighthouse does not complain, that files are not zipped. And also to give me more clearer picture, what production build size would look like.